src/js/components/search/table/ResultsTableTabItem.jsx

```diff
@@ -25,36 +26,36 @@ export default class ResultsTableTabItem extends React.Component {
     }
 
     render() {
-        let activeClass = '',
-            comingSoon = '',
-            status = '',
-            clickEvent = '';
+        let activeClass = '';
+        let comingSoon = '';
+        let comingSoonModule = (
```

Comment:
This needs to be `const` to pass the linting (getting a green checkmark in Github instead of the red X)